geotrellis/geotrellis-server
# Update dependencies

## Overview

This PR splits the cats ecosystem dependencies to match GeoTrellis 3.2

 - [x] TODO: Update MAML once https://github.com/geotrellis/maml/pull/112 is released -- using a snapshot until there's a release -- good enough for development for now

### Checklist


## Testing Instructions

- CI should hopefully be enough to verify everything compiles -- I reviewed `evicted` from the `sbt` shell too in order to make sure that between versions cats dependencies looked sane, but there's a lot of evictions
